File: C/010_projecteuler.c
#include <stdio.h>
#include <stdbool.h>
#include <math.h>

bool is_prime(int num) {
    for (int i = 2; i <= sqrt(num); i++) {
        if (num % i == 0) {
            return false;
        }
    }
    return true;
}

int main() {
    long sum_of_primes = 0;
    long limit = 2000000;

    for (int n = 2; n < limit; n++) {
        if (is_prime(n)) {
            sum_of_primes += n;
        }
    }

    printf("%ld\n", sum_of_primes);
    return 0;
}

File: python/010_projecteuler.py
import math

def is_prime (num):
    for i in range(2,int(math.sqrt(num))+1):
        if num % i == 0:
            return False
    return True

def main ():
    sum_of_primes = 0
    limit = 2000000
    for n in range(2, limit):
        if is_prime(n):
            sum_of_primes += n

    print(sum_of_primes)
main()
